airflow/configuration.py

```diff
@@ -76,8 +77,8 @@ def run_command(command):
     Runs command and returns stdout
     """
     process = subprocess.Popen(
-        command.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    output, stderr = process.communicate()
+        shlex.split(command), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    output, stderr = [stream.decode() for stream in process.communicate()]
```

Comment:
won't this raise an exception unnecessarily if `process` writes non-ascii bytes to its stdout/stderr?